README.md

```diff
@@ -7,7 +7,7 @@ This repo contains all the source code we use to build our docs site.
 
 ## We are Astro
 
-Astro is a site builder for the web platform.
+Astro is a website build tool for the modern web.
```

Comment:
```suggestion
Astro is an all-in-one web framework for building fast, content-focused websites.
```
Hey @MarcusOtter! Thanks for this, and you're right... what's there is pretty dated. 😅 

This is what we use in Docs itself, on our getting started page and in "Why Astro?" What do you think about copying this?

Reply:
I think that's a wonderful suggestion 🎉